libraries/app/database_api.cpp

```diff
@@ -1280,6 +1283,9 @@ market_ticker database_api_impl::get_ticker( const string& base, const string& q
       {
          orders = get_order_book(assets[0]->symbol, assets[1]->symbol, 1);
       }
+      if( to_subscribe )
+         subscribe_to_item( itr->id );
```

Comment:
I think this is not sufficient.

1. Currently we don't return `id` in `market_ticker`, so the client don't know what `id` is for what.

2. I guess, when the data changed, only the `id` will be pushed to the client, so the client need to call `get_objects` to get the new data. This is not good enough. We'd better push the data directly.